monitor/census.go

```diff
@@ -1869,6 +1874,14 @@ func (cen *censusMetricsCounter) recordAIRequestLatencyScore(pipeline string, Mo
 	if err := stats.RecordWithTags(cen.ctx, tags, cen.mAIRequestLatencyScore.M(latencyScore)); err != nil {
 		glog.Errorf("Error recording metrics err=%q", err)
 	}
+	SendQueueEvent("ai_request_started", map[string]string{
```

Comment:
Don't send events from `census.go`. Census should be related only to Prometheus metrics. If you want to do both (metrics and event) at the same time, please extract it to some helper function in a separate go file.